hubbard/hamiltonian.py

```diff
@@ -841,8 +841,8 @@ def DOS(self, egrid, eta=1e-3, spin=[0, 1], dist='Lorentzian', eref=0.):
             dos += sisl.electron.DOS(egrid, eig, distribution=dist)
         return dos
 
-    def PDOS(self, egrid, eta=1e-3, spin=[0, 1], dist='Lorentzian', eref=0.):
-        """ Obtains the projected density of states (PDOS) of the system with a distribution function
+    def LDOS(self, egrid, eta=1e-3, spin=[0, 1], dist='Lorentzian', eref=0.):
```

Comment:
I think we mistunderstood each other. What you have here _was_ correct. I.e. that was calculating the `PDOS`. So you can revert this change.

Reply:
Oh I see, I will revert this commit. Thanks!

Reply:
Done! This last commit was deleted and now HEAD is at  1d62f6519bf0b5f00417ba64d271b33ad2b1efb3